.laminas-ci.json

```diff
@@ -1,10 +1,45 @@
 {
     "ignore_php_platform_requirements": {
-        "8.1": true
+        "8.0": false,
```

Comment:
We can omit these here as with one of the latest versions of the matrix action I've changed the defaults of all versions to `false`

Reply:
fixed